os/board/rtl8720e/src/component/bluetooth/api/Make.defs

```diff
@@ -0,0 +1,55 @@
+###########################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
```

Comment:
Could you update the year of copyright for Samsung License? 2020 -> 2022

Reply:
Dear Mr. Chang, Sorry, I'll fix this.